test/testPlotWidget.cc

```
@@ -86,5 +87,447 @@ namespace minsky
+      box.x1 = 300;
+      box.y1 = 200;
+      
+      resize(box);
+      
+      EXPECT_EQ(300.0f, iWidth());
+      EXPECT_EQ(200.0f, iHeight());
+      EXPECT_EQ(150.0f, x());  // Center x = (0 + 300) / 2
+      EXPECT_EQ(100.0f, y());  // Center y = (0 + 200) / 2
+    }
+
+    TEST_F(PlotWidgetTest, clickTypeOnItem)
```

Clang (14.0.6)

[error] all parameters should be named in a function

(readability-named-parameter,-warnings-as-errors)